src/scss/config/_focus.scss

```diff
@@ -0,0 +1,12 @@
+// Focus
+// -----
+/// Consistent focus ring,
+/// is applied by default on element-focus,
+/// but some patterns (like radio-buttons) also require
+/// focus-ring applied to a sibling element.
+/// @group config-focus
```

Comment:
@mirisuzanne This is missing a group-mapping in `.sassdocrc`.

Reply:
@mirisuzanne Given your comment on the SVGs, I think this is the only remaining thing before merge?